internal/ci/ethereum_test

```diff
@@ -112,9 +112,13 @@ printf -- "\033[34mMigration complete.\033[0m\n"
 ## Run ChainLink
 ########################
 
+printf -- "\033[34mMaking chainlink...\033[0m\n"
+make install &>$SRCROOT/integration/make.log
+
 printf -- "\033[34mStarting chainlink...\033[0m\n"
-export CHAINLINK_DEV=true
-cldev &>$ROOT/integration/cldev.log &
+source $SRCROOT/internal/bin/clenv
+export ROOT=$SRCROOT/internal/clroot
```

Comment:
Had to rename `ROOT` to `SRCROOT` because `chainlink` uses `ROOT` to define our home directory, where we house bolt and logs.